src/smpp_gateway/admin.py

```diff
@@ -28,6 +29,26 @@ class MTMessageStatusInline(admin.TabularInline):
     extra = 0
 
 
+class CommandListListFilter(admin.SimpleListFilter):
+    title = "command status"
+    parameter_name = "command_status"
+
+    def lookups(self, request, model_admin):
+        """
+        Only show the lookups for command_status values that exist in the database.
+        """
+        statuses = MTMessageStatus.objects.values_list(
+            "command_status", flat=True
+        ).distinct()
+        return [(status, DESCRIPTIONS[status]) for status in statuses]
```

Comment:
```suggestion
        return [(status, DESCRIPTIONS.get(status, '')) for status in statuses]
```

Thinking on this, we should probably support the "we don't have that code" case